build.sh

```diff
@@ -60,17 +58,13 @@ function parse_arguments() {
             ORGANIZATION="$2"
             shift; shift;
             ;;
-            --latest-only)
-            LATEST_ONLY=true
-            shift
-            ;;
-            --use-digests)
-            USE_DIGESTS=true
```

Comment:
Digests are now on all the time? +1 if so.

Reply:
yes, all the time